packages/lab/src/pill/Pill.css

```diff
@@ -1,248 +1,89 @@
-  --saltButton-background-hover: var(--pill-background-hover);
-
-  --saltIcon-color: var(--pill-icon-color);
-  --saltIcon-color-hover: var(--pill-icon-color-hover);
-
-  background: var(--pill-background-hover);
-}
-
-/* Style applied to Pill when active */
-.saltPill:active,
-.saltPill-active {
-  --saltButton-background-active: var(--pill-background-active);
-}
-
-/* Style applied to Pill on focus */
-.saltPill:focus {
-  outline-color: var(--saltPill-focused-outlineColor, var(--salt-focused-outlineColor));
-  outline-style: var(--saltPill-focused-outlineStyle, var(--salt-focused-outlineStyle));
-  outline-width: var(--salt-focused-outlineWidth);
-  outline-offset: var(--salt-focused-outlineOffset);
-}
-
-/* Style applied to Pill icon if `clickable={true}`  */
-.saltPill-clickable {
-  --saltButton-background-active: var(--pill-background);
-  --saltButton-background-hover: var(--pill-background);
-
-  cursor: var(--salt-selectable-cursor-hover);
-  user-select: none;
-}
-
-/* Style applied to Pill when `deletable={true}` */
-.saltPill-deletable {
-  --saltButton-background-active: var(--pill-background);
-  --saltButton-background-hover: var(--pill-background);
-}
-
-.saltPill-selectable:hover,
-.saltPill-deletable:hover {
-  --saltButton-background-active: var(--pill-background);
-  --saltButton-background-hover: var(--pill-background);
-}
-
-/* Style applied to Pill when active if `selectable={true}` */
-.saltPill-selectable:active,
-.saltPill-active.saltPill-selectable {
-  --saltButton-background-active: var(--pill-background);
-}
-
-/* Style applied to Pill when disabled */
-.saltPill-disabled,
-.saltPill-disabled:hover,
-.saltPill-disabled:active {
-  --saltButton-background-active: var(--pill-background-disabled);
-  --saltButton-background-hover: var(--pill-background-disabled);
-  --saltButton-text-color-active: var(--pill-text-color);
-  --saltButton-text-color-hover: var(--pill-text-color);
-  --saltCheckbox-icon-tick-color: var(--salt-taggable-foreground-disabled);
-  --saltCheckbox-icon-stroke-hover: var(--salt-selectable-borderColor-disabled);
-
-  background: var(--pill-background-disabled);
-  cursor: var(--salt-selectable-cursor-disabled);
-}
-
-/* Checkbox styles for selectable Pill when `disabled={true}` */
-.saltPill-disabled .saltPill-checkbox {
-  --saltCheckbox-icon-tick-color: var(--salt-taggable-foreground-disabled);
-  --saltCheckbox-icon-stroke: var(--salt-selectable-borderColor-disabled);
-  --saltCheckbox-icon-stroke-hover: var(--salt-selectable-borderColor-disabled);
+  gap: var(--saltPill-gap, var(--salt-spacing-50));
+  padding-inline: var(--salt-spacing-50);
+  color: var(--saltPill-text-color, var(--pill-text-color));
+  font-size: var(--saltPill-fontSize, var(--salt-text-fontSize));
+  font-weight: var(--saltPill-fontWeight, var(--salt-text-fontWeight));
+  letter-spacing: var(--saltPill-letterSpacing, 0);
+  line-height: var(--saltPill-lineHeight, var(--salt-text-label-lineHeight));
```

Comment:
you seem to have a lot of vars for customisation. Is this intended? Has it been discussed with design?